### PR TITLE
Flow tutorial doc update mk2

### PR DESCRIPTION
**Pull request by pollett**
Clean copy of @amandalindsay 's https://github.com/hazelcast/hz-flow-docs/pull/79

### CODE REVIEW

**Review by pmcgleenon — APPROVED** [review]

**Review by pawelklasa — APPROVED** [review]

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 1]
```diff
@@ -28,15 +25,22 @@ operation findAwards(FilmId):Award[]
 
 image:2architecture-overview.png[The services in our demo]
 
-We'll stitch these together, and expose a REST API that returns the data from all three services.
+We'll combine these together to create a single endpoint that exposes a REST API which returns the data from all three services.
 
 == Set up
```
Comment (Rob-Hazelcast) [inline docs/modules/guides/pages/work-with-xml.adoc:30]: Extremely nitpicky, but I think this should be Setup or Set up [something]

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 2]
```diff
@@ -28,15 +25,22 @@ operation findAwards(FilmId):Award[]
...
+Create a new Taxi project, for example, called `xml-and-json`. 
+
+With {short-product-name}, instead of using integration code, schemas are used to describe  data sources which {short-product-name} uses to link everything together. Some services publish their own schemas (e.g. XSDs or WSDLs) but the following example shows how to do this without schemas, with everything written in Taxi. If you have existing schemas, you can skip these steps. 
 
 == Describe the Film Service
 
-Write some Taxi that describes our Film service:
+First, we'll write Taxi code that describes our Film service. This is an HTTP operation that returns a list of films.
+
+In the Taxi model, you'll notice  that the name of the field is comprised of special types which will be used to share data across services, and connect them together.
```
Comment (Rob-Hazelcast) [inline docs/modules/guides/pages/work-with-xml.adoc:42]:
```suggestion
In the Taxi model, you'll notice that the name of the field is composed of special types that will be used to share data across services and connect them together. 
```

"comprises" or "is composed of", if we're being strict

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 3]
```diff
@@ -182,7 +189,7 @@ Which returns:
 
 === Restructure the result
 
-We'd like to remove the `item` wrapper (which is carried over from the XML format), so we change the query, to ask just for a `Film[]`
+To remove the `item` wrapper (which is carried over from the XML format), we can change the query to just ask for a `Film[]`
```
Comment (Rob-Hazelcast) [inline docs/modules/guides/pages/work-with-xml.adoc:192]:
```suggestion
To remove the `item` wrapper (which is carried over from the XML format), we can change the query to just ask for a `Film[]`:
```

**Review by Rob-Hazelcast — COMMENTED** [review]
A few nitpicky comments (apologies), but generally looks good. Decent structure, nice diagram, well-formatted examples etc. Several comments are suggestions, which you can batch up and apply in the GitHub UI. Keep in mind I'm not familiar with these docs or this product, so feel free to ignore anything that's off the mark.

There's less hand-holding than I'm used to seeing, but maybe that's because we expect the reader to have a high level of knowledge, or because this is a tutorial walking through an example rather than a procedure giving instructions, or both.

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 4]
```diff
@@ -28,15 +25,22 @@ operation findAwards(FilmId):Award[]
...
 
 == Describe the Film Service
 
-Write some Taxi that describes our Film service:
+First, we'll write Taxi code that describes our Film service. This is an HTTP operation that returns a list of films.
+
+In the Taxi model, you'll notice  that the name of the field is comprised of special types which will be used to share data across services, and connect them together. 
+This means we are not coupled to field names, which are free to change without impacting our model. 
 
 [tabs]
```
[the remark this answers is not in the document]
Comment (oliverhowell) [reply docs/modules/guides/pages/work-with-xml.adoc:45]: Yeah, same experience but also like the compactness. I think these samples need more explicit signposting and even numbered steps as Rob notes earlier. If user has to create the 1st file and gets the 2nd file as a result we should be explicit about that, especially when we're using these code tabs in a different way (which I'm ok with if it coheres and is easy to understand).

File: docs/modules/guides/pages/work-with-xml.adoc [hunk 5]
```diff
@@ -28,15 +25,22 @@ operation findAwards(FilmId):Award[]
...
 == Set up
 
-To get started working on {short-product-name}, follow the instructions at xref:deploy:development-deployments.adoc[Development Quickstart guide]
+To get started with {short-product-name}, follow the instructions at xref:deploy:development-deployments.adoc[Developer setup].
+
+Create a new Taxi project, for example, called `xml-and-json`. 
+
+With {short-product-name}, instead of using integration code, schemas are used to describe  data sources which {short-product-name} uses to link everything together. Some services publish their own schemas (e.g. XSDs or WSDLs) but the following example shows how to do this without schemas, with everything written in Taxi. If you have existing schemas, you can skip these steps. 
 
 == Describe the Film Service
```
[the remark this answers is not in the document]
Comment (oliverhowell) [reply docs/modules/guides/pages/work-with-xml.adoc:38]: Sentence case!

**Pull request merged**